lisa/sut_orchestrator/hyperv/context.py

```diff
@@ -1,21 +1,37 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
-from dataclasses import dataclass
+from dataclasses import dataclass, field
 from pathlib import PurePath
-from typing import Optional
+from typing import List, Optional
 
 from lisa import Node, RemoteNode
+from lisa.sut_orchestrator.hyperv.schema import DeviceAddressSchema
+from lisa.sut_orchestrator.util.schema import HostDevicePoolType
 from lisa.util.process import Process
 
 
+@dataclass
+class DevicePassthroughContext:
+    pool_type: HostDevicePoolType = HostDevicePoolType.PCI_NIC
+    device_list: List[DeviceAddressSchema] = field(
+        default_factory=list,
+    )
+
+
 @dataclass
 class NodeContext:
     vm_name: str = ""
     host: Optional[RemoteNode] = None
     working_path = PurePath()
     serial_log_process: Optional[Process] = None
 
+    # Device pass through configuration
+    is_device_passthrough_set: bool = False
```

Comment:
Is it a setting in the HyperV server?

Reply:
Removed. All conditions are now on list length